fecgov/openFEC
# Feature/cookie settings

## Summary

- Adding rules for the Google Analytics cookie
- Updating the GA ID from 18F's
- Page code formatting (indents)

## Impacted areas of the application

Technically every page but could only affect GA functionality

## Reviewers

TBD?

## Screenshots

None

## Related PRs

None

## How to test
- May need to test on a dev space
- Check Inspector to see that the FEC GA cookie's settings are correct

____
